_posts/2017-05-23-0.1-announcement.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Introducing Istio: A robust service mesh for microservices"
+overview: Istio 0.1 announcement
+published: true
+permalink: blog/istio-service-mesh-for-microservices.html
+attribution: The Istio Team
+layout: post
+type: markdown
+---
+
+Google, IBM, and Lyft are proud to announce the first public release of [Istio](https://istio.io/): an open source project that provides a uniform way to connect, secure, manage and monitor microservices. Our current release is targeted at the [Kubernetes](https://kubernetes.io/) environment; we intend to add support for other environments such as virtual machines and Cloud Foundry in the coming months.
```

Comment:
"Our current release..." that whole sentence should be down low in the doc. It should most certainly not be the second sentence of an announcement.